edgelet/docker-rs/src/models/container_create_body.rs

```diff
@@ -58,7 +58,7 @@ pub struct ContainerCreateBody {
     #[serde(rename = "Image", skip_serializing_if = "Option::is_none")]
     image: Option<String>,
     #[serde(rename = "Volumes", skip_serializing_if = "Option::is_none")]
-    volumes: Option<::models::ContainerConfigVolumes>,
+    volumes: Option<Value>,
```

Comment:
It feels like this should be Option<::std::collections::HashMap<String, Value>>

Reply:
Yes, changed it.